cli/flox/src/commands/list.rs

```diff
@@ -65,7 +67,8 @@ impl List {
 
         let manifest_contents = env.manifest_contents(&flox)?;
         if self.list_mode == ListMode::Config {
-            tracing::Span::current().record("mode", "config");
+            sentry_set_tag("mode", "config");
+
```

Comment:
did you mean to drop the tags for the other modes below?

Reply:
Bah, I meant to drop all of them, now that they're set at the top of the function. This snuck back in during a rebase 🙈